src/routes/InventoryDetail.js

```diff
@@ -75,6 +75,11 @@ const Inventory = () => {
     const writePermissions = useWritePermissions();
     const entityLoaded = useSelector(({ entityDetails }) => entityDetails?.loaded);
     const entity = useSelector(({ entityDetails }) => entityDetails?.entity);
+    const cloudProvider = useSelector(({ systemProfileStore }) => systemProfileStore?.systemProfile?.cloud_provider);
+    const availableApps = useMemo(() => appList.map((app) => app.name === 'ros' ? {
+        ...app,
+        isVisible: ['aws', 'azure'].includes(cloudProvider)
```

Comment:
@karelhala ATM ros is only supported for aws, can you please update this accordingly!